app/src/main/kotlin/cz/covid19cz/erouska/ui/dashboard/DashboardFragment.kt

```diff
@@ -41,6 +42,10 @@ class DashboardFragment : BaseFragment<FragmentDashboardPlusBinding, DashboardVM
 
     companion object {
         private const val SCREEN_NAME = "Dashboard"
+
+        private const val ANALYTICS_KEY_SHARE_APP = "click_share_app"
```

Comment:
Can you document all Analytics events somewhere? Wiki page here on GitHub is probably the best place.